server/lib/buff_server_grpc/auth_service/server.ex

```diff
@@ -0,0 +1,20 @@
+defmodule BuffServerGrpc.AuthService.Server do
+  use GRPC.Server, service: BuffServerGrpc.AuthService.Service
+
+  @login_error GRPC.RPCError.exception(
+                 GRPC.Status.unauthenticated(),
+                 "Incorrect credentials"
+               )
+  def login(%{username: username, password: password}, _stream) do
```

Comment:
I know you've probably given this a lot of thought, but why not simply use email instead of username?

Reply:
not that mich, good discussion to have.
i personally like nicknames.. 
the only real reason i can think of is that you might not want to expose your email.. thats why i also split between public email and private.. but maybe there’s a better way of doing that

Reply:
I've done some reading about this.
i agree. i'll merge this PR and do another refactor.